app/views/search/search.html.erb

```diff
@@ -0,0 +1,180 @@
+                                ["1 Guest", 1],
+                                ["2 Guests", 2],
+                                ["3 Guests", 3],
+                                ["4+ Guests", 4]
+                            ], selected: f.object.accommodate), include_blank: true, class: "col-12" %>
+                </div>
+                <div class="form-group mx-5">
+                    <h6>Bedroom</h6>
+                    <%= f.select :bedroom, options_for_select([
+                                ["1 Guest", 1],
+                                ["2 Guests", 2],
+                                ["3 Guests", 3],
+                                ["4+ Guests", 4]
+                            ], selected: f.object.bedroom), include_blank: true, class: "col-12" %>
+                </div>
+                <div class="form-group mx-5">
+                    <h6>Bathroom</h6>
+                    <%= f.select :bathroom, options_for_select([
+                                ["1 Guest", 1],
+                                ["2 Guests", 2],
+                                ["3 Guests", 3],
+                                ["4+ Guests", 4]
+                            ], selected: f.object.bathroom), include_blank: true, class: "col-12" %>
+                </div>
+            </div>
+           <% end %>
+        </div>
+    </div>
+    <div class="border p-3">
+        <form class="form-inline">
+            <h5>Amenities</h5>
+            <div class="form-group ml-4">
+                <%= search_form_for @search, url: search_path do |f| %>
+                    <div class="form-check ml-5">
+                        <%= check_box_tag "q[internet_eq]" ,true %>
+                        <label class="form-check-label" for="formCheck-1">TV</label>
+                    </div>
+                    <div class="form-check ml-5">
+                        <%= check_box_tag "q[internet_eq]" ,true %>
+                        <label class="form-check-label" for="formCheck-1">Kitchen</label>
+                    </div>
+                    <div class="form-check ml-5">
+                        <%= check_box_tag "q[internet_eq]" ,true %>
+                        <label class="form-check-label" for="formCheck-1">Internet</label>
+                    </div>
+                    <div class="form-check ml-5">
+                        <%= check_box_tag "q[internet_eq]" ,true %>
+                        <label class="form-check-label" for="formCheck-1">Heating</label>
+                    </div>
+                    <div class="form-check ml-5">
+                        <%= check_box_tag "q[internet_eq]" ,true %>
+                        <label class="form-check-label" for="formCheck-1">Air conditioning</label>
+                    </div>
+                <% end %>
+            </div>
+        </form>
+    </div>
+    <%= search_form_for @search, url: search_path do |f| %>
+        <%= link_to "Search", search_path, class: "btn btn-outline-dark my-2 my-sm-0 offset-11" %>
```

Comment:
Advanced search filter is not working..

try deleting _Line 156_ and change _Line 157_ to a submit button